pages/flow/index.js

```diff
@@ -1,22 +1,48 @@
 import styled from "styled-components";
 import AsanaCard from "../../components/Asana-Card/AsanaCard";
 import { getAsanaByID } from "../../db";
-import { useState } from "react";
+import asanas from "../../db";
 
-export default function Flow() {
-  const [flowAsanaIds, setFlowAsanaIds] = useState(["5", "2", "3", "4"]);
-
-  const flowAsanas = flowAsanaIds.map((item) => {
-    return getAsanaByID(item);
+export default function Flow({
+  flowAsanaIds,
+  setFlowAsanaIds,
+  selectedAsanaIds,
+  setSelectedAsanaIds,
+}) {
+  const flowAsanas = flowAsanaIds.map((id) => {
+    return getAsanaByID(id);
   });
 
   return (
     <>
-      <h2>Your flow for today:</h2>
+      <StyledH2>Choose your Asanas for today:</StyledH2>
+      <AddAsanaSection>
+        <StyledList>
+          {asanas.map((asana) => {
+            return (
```

Comment:
Ich glaube du brauchst hier gar kein return. Check das mal bitte. Es gab bei einer Ähnlichen Konstellation bei mir nämlich ab und zu dann Crashs.

Reply:
Yes, this should work as an implicit return as well.

Reply:
Du hast Recht. Danke für den Tip!